sdk/src/main/java/com/bugsnag/android/NativeInterface.java

```diff
@@ -6,6 +6,7 @@
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 
+import java.nio.charset.StandardCharsets;
```

Comment:
This was added in API 19, so we'll need to use `Charset.forName("UTF-8")` instead to support API 14+